EventVisualisation/DataConverter/include/EventVisualisationDataConverter/VisualisationTrack.h

```diff
@@ -105,22 +110,23 @@ class VisualisationTrack
   void addMomentum(const double pxpypz[3]);
 
   int mID;                     /// Unique identifier of the track
-  std::string mType;           /// Type (standard, V0 mother, daughter etc.)
+                               //  std::string mType;           /// Type (standard, V0 mother, daughter etc.)
+  float mTime;                 /// track time
   int mCharge;                 /// Charge of the particle
-  double mEnergy;              /// Energy of the particle
-  int mParentID;               /// ID of the parent-track (-1 means no parent)
+                               //  double mEnergy;              /// Energy of the particle
```

Comment:
if the commented out code is obsolete, better to remove it.

Reply:
I remove it (but it return in a future)